src/Workspaces/MSBuildTest/Utilities/VisualStudioMSBuildInstalled.cs

```diff
@@ -3,46 +3,48 @@
 // See the LICENSE file in the project root for more information.
 
 using System;
-using System.Linq;
 using Microsoft.Build.Locator;
 using Roslyn.Test.Utilities;
 
 namespace Microsoft.CodeAnalysis.MSBuild.UnitTests
 {
     internal class VisualStudioMSBuildInstalled : ExecutionCondition
     {
+        private readonly Version _minimumVersion;
+
 #if NET472_OR_GREATER
+
         private static readonly VisualStudioInstance? s_instance;
```

Comment:
```suggestion
#if NET472_OR_GREATER

        private static readonly VisualStudioInstance? s_instance;
        private readonly Version _minimumVersion;
```

Reply:
I tried moving this out originally but I see it's breaking the build in other fun ways, so back it goes...